source/extensions/transport_sockets/tls/cert_validator/spiffe/spiffe_validator.cc

```diff
@@ -165,9 +165,9 @@ int SPIFFEValidator::doVerifyCertChain(X509_STORE_CTX* store_ctx,
   bssl::UniquePtr<X509_STORE_CTX> verify_ctx(X509_STORE_CTX_new());
   // We make a copy of X509_VERIFY_PARAMs in the store_ctx that we received as a parameter.
   // This is a precaution mostly, as Envoy doesn't configure any X509_VERIFY_PARAMs.
-  // Note that there's no api to copy crls from one store_ctx to another; the assumption is that
-  // neither X509_V_FLAG_CRL_CHECK nor X509_V_FLAG_CRL_CHECK_ALL verify_params are not used.
-  // Should this change, consider opening up X509_STORE_CTX struct, which is internal atm.
+  // Note that there's no api to copy crls from one store_ctx to another; the assumption is that 
```

Comment:
Are we sure we included the rest of Marko his fix for this validation?

Reply:
Yes, it was only the change to the comment that didn't make it to 2.2